codechecker_lib/analyzers/result_handler_plist_to_db.py

```diff
@@ -190,3 +165,9 @@ def handle_results(self):
             self.__store_bugs(files, bugs, connection, analysis_id)
 
             connection.finish_build_action(analysis_id, self.analyzer_stderr)
+
+    def postprocess_result(self):
```

Comment:
I'm not sure why is this here better than in a base class with a default behaviour doing nothing?
It is just duplicated in the result_handler_plist_to_stdout.py module.